models/fastText_expirements.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
<p>Do you need this our can you just write FirstColFtClassifier() directly into the function in the cell below. </p><p> </p><p>How did you decide that lr=0.3 and epoch=10 were the right parameters? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I chose it arbitrarily from a tutorial, but you're right on the first point that I don't need the extra step and I may as well use the default parameters. Choosing the right parameters here requires some thought and tuning so I'll hold off on that for a bit